app/views/qsets/index.html.erb

```diff
@@ -1,8 +1,4 @@
-<div class="container">
-  <br>
-  <h3 class="center"><b>Question Sets</b></h3><br>
-
-  <div class="row qset_list">
+   <div class="row qset_list">
```

Comment:
Given it's a partial, we can start the indenting at column 0, no need to push everything to the right like this.